department-of-veterans-affairs/va.gov-cms
# VACMS-4846 No content type should show Submitted or Owner info

## Description
See #4846

## Testing done
Visual / Behat

## Screenshots
![image](https://user-images.githubusercontent.com/2404547/117674131-6777dd80-b179-11eb-8d14-19d24ba2bbbc.png)

## QA steps
As user Victor.A.MCtest

1. Go to https://pr5277-8r6jpjic9gjyuubnmuh5m98ivg4m6oxh.ci.cms.va.gov/escanaba-vet-center/community-access-point/escanaba-vet-center-houghton
2. You should not see node author and date.

1. Go to https://pr5277-8r6jpjic9gjyuubnmuh5m98ivg4m6oxh.ci.cms.va.gov/escanaba-vet-center/community-access-point/escanaba-vet-center-houghton
2. You should not see Owner field (excluding right rail block top block)

1. Search all files with this pattern: `config/sync/node.type.*.yml` - confirm they all show `display_submitted: false`
2. Search all files with this pattern: `core.entity_view_display.node.*.yml` - confirm they all show `hidden: field_administration: true`

### Definition of Done


### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
